generic_tracers/generic_COBALT.F90

```diff
@@ -1898,4197 +1898,7 @@ subroutine generic_COBALT_register_diag(diag_list)
-         init_time, vardesc_temp%longname,vardesc_temp%units, missing_value = missing_value1, &
-         cmor_field_name="phypicoos", cmor_units="mol m-3",                          &
-         cmor_standard_name="mole_concentration_of_picophytoplankton_expressed_as_carbon_in_sea_water", &
-         cmor_long_name="Surface Mole Concentration of Picophytoplankton expressed as Carbon in sea water")
-
-    vardesc_temp = vardesc("phymiscos_raw","Surface Mole Concentration of Miscellaneous Phytoplankton expressed as Carbon in sea water",'h','1','s','mol m-3','f')
-    cobalt%id_phymiscos = register_diag_field(package_name, vardesc_temp%name, axes(1:2), &
-         init_time, vardesc_temp%longname,vardesc_temp%units, missing_value = missing_value1, &
-         cmor_field_name="phymiscos", cmor_units="mol m-3",                          &
-         cmor_standard_name="mole_concentration_of_miscellaneous_phytoplankton_expressed_as_carbon_in_sea_water", &
-         cmor_long_name="Surface Mole Concentration of Miscellaneous Phytoplankton expressed as Carbon in sea water")
-
-    vardesc_temp = vardesc("zmicroos_raw","Surface Mole Concentration of Microzooplankton expressed as Carbon in sea water",'h','1','s','mol m-3','f')
-    cobalt%id_zmicroos = register_diag_field(package_name, vardesc_temp%name, axes(1:2), &
-         init_time, vardesc_temp%longname,vardesc_temp%units, missing_value = missing_value1, &
-         cmor_field_name="zmicroos", cmor_units="mol m-3",                          &
-         cmor_standard_name="mole_concentration_of_microzooplankton_expressed_as_carbon_in_sea_water", &
-         cmor_long_name="Surface Mole Concentration of Microzooplankton expressed as Carbon in sea water")
-
-    vardesc_temp = vardesc("zmesoos_raw","Surface Mole Concentration of Mesozooplankton expressed as Carbon in sea water",'h','1','s','mol m-3','f')
-    cobalt%id_zmesoos = register_diag_field(package_name, vardesc_temp%name, axes(1:2), &
-         init_time, vardesc_temp%longname,vardesc_temp%units, missing_value = missing_value1, &
-         cmor_field_name="zmesoos", cmor_units="mol m-3",                          &
-         cmor_standard_name="mole_concentration_of_mesozooplankton_expressed_as_carbon_in_sea_water", &
-         cmor_long_name="Surface Mole Concentration of Mesozooplankton expressed as Carbon in sea water")
-
-    vardesc_temp = vardesc("talkos_raw","Surface Total Alkalinity",'h','1','s','mol m-3','f')
-    cobalt%id_talkos = register_diag_field(package_name, vardesc_temp%name, axes(1:2), &
-         init_time, vardesc_temp%longname,vardesc_temp%units, missing_value = missing_value1, &
-         cmor_field_name="talkos", cmor_units="mol m-3",                          &
-         cmor_standard_name="sea_water_alkalinity_expressed_as_mole_equivalent", &
-         cmor_long_name="Surface Total Alkalinity")
-
-    vardesc_temp = vardesc("talknatos_raw","Surface Natural Total Alkalinity
```

Comment:
I suggest we change the file extension to `inc`. Please see my comments in the Python script for the reason.